src/managers/config.py

```diff
@@ -42,13 +45,28 @@
 auto.create.topics.enable=false
 metric.reporters=com.linkedin.kafka.cruisecontrol.metricsreporter.CruiseControlMetricsReporter
 """
+TESTING_OPTIONS = """
+cruise.control.metrics.reporter.metrics.reporting.interval.ms=6000
+"""
```

Comment:
**question:** This is the same as in `CRUISE_CONTROL_TESTING_OPTIONS`, is that intentional?

Reply:
Yes, this is intentional. I remember having an error message when the option was not set on both sides